pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -68,7 +68,6 @@ stdenv.mkDerivation rec {
     cd "$out/etc/fonts"
     rm conf.d/{50-user,51-local}.conf
```

Comment:
I see, thanks for the answer.

After looking more at it, `51-local.conf` is removed because it is the file we want to generate via the module (`30-nixos-generic-alias.conf` in this PR).

I guess `50-user.conf` is removed and generated in the module for interaction with `includeUserConf` setting, but that could be improved.

I will look if a commit can be added to make this PR closer to upstream definitions.